pages/platform/kubernetes-k8s/known-limits/guide.en-asia.md

```diff
@@ -104,11 +103,36 @@ In any case, there are some ports that you shouldn't block on your instances if
 - UDP Port 123 (*systemd-timesync*): needed to allow NTP servers synchronization
 - TCP/UDP Port 53 (*systemd-resolve*): needed to allow domain name resolution
 - TCP Port 111 (*rpcbind*): needed only if you want to use the NFS client deployed on nodes managed by OVHcloud
+- TCP Port 4443 (metrics server): needed to list the metrcis of the nodes and pods
 
 ### Ports to open from others worker nodes (INPUT/OUPUT)
 
 - UDP Port 8472 (*flannel*): needed for communication between pods
 - UDP Port 4789 (*kube-dns internal usage*): needed for DNS resolution between nodes
+- TCP Port 10250 (*kubelet*): needed for [communication between apiserver and worker nodes](https://kubernetes.io/docs/concepts/architecture/master-node-communication/#apiserver-to-kubelet)
+
+### Openstack security group point of view
+
+In case you want to add Openstack security group on your nodes, it is mendatory to add the above ports in the ruleset with `0.0.0.0/24 CIDR`.
```

Comment:
```suggestion
In case you want to apply OpenStack security groups onto your nodes, it is mandatory to add the above ports in a ruleset concerning the `0.0.0.0/24` CIDR.
```